base/reflection.jl

```diff
@@ -1350,8 +1350,8 @@ function _which(@nospecialize(tt::Type), world=get_world_counter())
     min_valid = RefValue{UInt}(typemin(UInt))
```

Comment:
Maybe this function should just reach into Core.Compiler and call findsup? I don't usually like having the internal ccalls in too many places.